diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index 8c750df..ff49233 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -1567,7 +1567,8 @@ a8xx_830 = GPUProps(
 # Totally fake, just to get cffdump to work:
 add_gpus([
         GPUId(chip_id=0x44050000, name="FD830"),
-        GPUId(chip_id=0x44050001, name="FD830")
+        GPUId(chip_id=0x44050001, name="FD830"),
+        GPUId(chip_id=0xffff44050001, name="FD830")
     ], A6xxGPUInfo(
         CHIP.A8XX,
         [a7xx_base, a7xx_gen3, a8xx_base, a8xx_830],
