diff --git a/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts b/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
index 8ed13c7..1e5d150 100644
--- a/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
+++ b/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
@@ -870,6 +870,21 @@ qup_uart15_cts_rts: qup-uart15-cts-rts-state {
        };
 };

+&remoteproc_adsp {
+        firmware-name = "qcom/sm8750/adsp.mbn",
+                        "qcom/sm8750/adsp_dtb.mbn";
+
+        status = "okay";
+};
+
+&remoteproc_cdsp {
+        firmware-name = "qcom/sm8750/cdsp.mbn",
+                        "qcom/sm8750/cdsp_dtb.mbn";
+
+        status = "okay";
+};
+
+
 /* Disabled features */
 // Mostly regulators that don't exist on the Odin 3
 // pm8010@d is disabled
