diff --git a/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts b/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
index 1e5d150..c04ce0c 100644
--- a/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
+++ b/arch/arm64/boot/dts/qcom/sm8750-ayn-odin3.dts
@@ -733,6 +733,37 @@ volume_up_n: volume-up-n-state {
        };
 };

+&pm8550_pwm {
+    status = "okay";
+
+    power-led {
+        label = "power-led";
+        color = <LED_COLOR_ID_RGB>;
+        function = LED_FUNCTION_STATUS;
+
+        #address-cells = <1>;
+        #size-cells = <0>;
+
+        led@1 {
+            reg = <1>;
+            label = "red";
+            color = <LED_COLOR_ID_RED>;
+        };
+
+        led@2 {
+            reg = <2>;
+            label = "green";
+            color = <LED_COLOR_ID_GREEN>;
+        };
+
+        led@3 {
+            reg = <3>;
+            label = "blue";
+            color = <LED_COLOR_ID_BLUE>;
+        };
+    };
+};
+
 &pon_pwrkey {
        status = "okay";
 };